### PR TITLE
Add Flower to the Mapping Glossary 

### PR DESCRIPTION
**Pull request by Jd-Gamer65**
Added beat saber flower to glossary.

### CODE REVIEW

File: wiki/mapping/glossary.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ let us know in #mapping-discussion!
 | **First Person Flying Controller** | A launch option available to SteamVR and Oculus users to launch Beat Saber with limited keyboard and mouse control over the game in a first person view. Very helpful for getting an accurate preview of [Lighting](#l) or [Walls](#wxyz), or accessing the official [Editor](#e). Abbreviated as [FPFC](#f). |
 | **Fixed BPM** | A song with a consistent [BPM](#b) from start to finish with no variation. Also known as [Single BPM](#s). See also: [Variable BPM](#uv). |
 | **Flick** | A [Pattern](#p) of two or more [Notes](#n) of the same color, typically at 1/4 [Precision](#p). <details><summary>**Example Image**</summary>![Picture of flick](~@images/mapping/flick-alt.png)</details> |
+| **Flower** | A combination of two or more [Notes](#n) that are different directions, creating a flower like shape.
```
Comment (bloodcloak) [inline wiki/mapping/glossary.md:92]: Could you provide an image containing a few examples in editor?

**Review by bloodcloak — CHANGES_REQUESTED** [review]

File: wiki/mapping/glossary.md (hunk 1 above)
Comment (Jd-Gamer65) [reply wiki/mapping/glossary.md:92]: Sure, attatch it here or in the repo?

Comment (bloodcloak) [reply wiki/mapping/glossary.md:92]: either is fine

Comment (Jd-Gamer65) [reply wiki/mapping/glossary.md:92]: Alright, Sorry it took me so long, GitHub did not want to cooperate.
Screenshot (8)

**Review by bloodcloak — APPROVED** [review]

**Pull request merged**